src/gui/mzroll/projectsaveworker.cpp

```diff
@@ -89,7 +89,9 @@ void TempProjectSaveWorker::saveProject(const bool saveRawData)
     if (_currentProjectName.isEmpty()) {
         auto now = QDateTime::currentDateTime();
         auto tempFileName = now.toString("dd_MM_yyyy_hh_mm_ss") + ".emDB";
-        auto firstSampleFile = _mw->getSamples()[0]->fileName;
+        string firstSampleFile = "";
+        if (!_mw->getSamples().empty())
+            auto firstSampleFile = _mw->getSamples()[0]->fileName;
```

Comment:
You declared a new variable by the name of `firstSampleFile`, therefore the one in outer scope was not overwritten. This is also why autosave now fails because it tries to write a temporary file in root (`/`) directory. Check your terminal logs.

Reply:
Made the correction.

Reply:
Oh your last commit overwrote this change. Ignore.